ezgripper_dual_gen1:

  controller_manager:
    ros__parameters:
      update_rate: 1000  # Hz

      ezgripper_controller:
        type: position_controllers/GripperActionController

  ezgripper_controller:
    ros__parameters:
      joint: left_ezgripper_knuckle_1
      max_effort: 1.0
